src/pages/Detail/Aside/AsideInfo.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import * as S from "./AsideInfo.style";
+
+export const AsideInfo = () => {
+  const [lecture, setLecture] = useState([]);
+
+  useEffect(() => {
+    fetch("/data/Detail/getDetail.json")
+      .then(res => res.json())
+      .then(data => setLecture(data.data));
+  }, []);
+  if (!lecture.lectureId) return null;
+
+  const star = lecture.review;
+
+  const reviewStar = [];
+  for (let data of star) {
+    reviewStar.push(data.reviewStar);
+  }
+
+  const avgStar = reviewStar.reduce((accumulator, current, index) => {
+    if (index === reviewStar.length - 1) {
+      return ((accumulator + current) / reviewStar.length).toFixed(1);
+    }
+    return accumulator + current;
+  }, 0);
+
+  const FormatPrice = Math.floor(lecture.lecturePrice)
```

Comment:
```suggestion
  const formatPrice = Math.floor(lecture.lecturePrice)
```
작명 컨벤션 한번더 체크하시면 완성도를 조금 더 높일 수 있다는 사실~